test/Microsoft.Azure.Devices.Edge.Test/Context.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test
+{
+    using System;
+    using System.IO;
+    using System.Net;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Configuration;
+
+    public class Context
+    {
+        static readonly Lazy<Context> Default = new Lazy<Context>(
+            () =>
+            {
+                IConfiguration context = new ConfigurationBuilder()
+                    .SetBasePath(Directory.GetCurrentDirectory())
+                    .AddJsonFile("context.json")
+                    .AddEnvironmentVariables("E2E_")
+                    .Build();
+
+                string GetString(string name) => context.GetValue<string>(name);
+
+                string GetStringOr(string name, string alternate)
```

Comment:
I think IConfiguration already has an overload that does this..

Reply:
You're probably right. I'll look...

Reply:
Yep. Fixed